napari/layers/base/base.py

```diff
@@ -782,27 +782,58 @@ def _extent_data(self) -> np.ndarray:
         """
         raise NotImplementedError
 
+    @property
+    def _extent_data_augmented(self) -> np.ndarray:
+        """Extent of layer in data coordinates.
+
+        Returns
+        -------
+        extent_data : array, shape (2, D)
+        """
+        return self._extent_data
+
     @property
     def _extent_world(self) -> np.ndarray:
         """Range of layer in world coordinates.
 
+        Returns
+        -------
+        extent_world : array, shape (2, D)
+        """
+        # Get full nD bounding box
+        return get_extent_world(self._extent_data, self._data_to_world)
+
+    @property
+    def _extent_world_augmented(self) -> np.ndarray:
+        """Range of layer in world coordinates.
+
         Returns
         -------
         extent_world : array, shape (2, D)
         """
         # Get full nD bounding box
         return get_extent_world(
-            self._extent_data, self._data_to_world, self._array_like
+            self._extent_data_augmented, self._data_to_world
         )
 
     @cached_property
     def extent(self) -> Extent:
         """Extent of layer in data and world coordinates."""
         extent_data = self._extent_data
         data_to_world = self._data_to_world
-        extent_world = get_extent_world(
-            extent_data, data_to_world, self._array_like
+        extent_world = get_extent_world(extent_data, data_to_world)
+        return Extent(
+            data=extent_data,
+            world=extent_world,
+            step=abs(data_to_world.scale),
         )
+
+    @cached_property
+    def extent_augmented(self) -> Extent:
+        """Extent of layer in data and world coordinates."""
```

Comment:
Describe how this is intended to be different than `extent` for layers in general.